components/session/share-session-modal.tsx

```
@@ -191,15 +181,13 @@ export default function ShareSessionModal({
                 className="space-y-4"
               >
                 <div className="space-y-2">
-                  <label className="text-sm font-semibold text-pink-400">
-                    Session Code:
-                  </label>
+                  <label className="text-sm font-semibold text-pink-400">Session Code:</label>
                   <div className="flex gap-2">
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)